consensus/src/consensus.rs

```diff
@@ -63,23 +65,49 @@ impl Consensus {
         )
         .await;
 
-        let mut core = Core::new(
-            name,
-            committee,
-            parameters,
-            signature_service,
-            store,
-            leader_elector,
-            mempool_driver,
-            synchronizer,
-            /* core_channel */ rx_core,
-            /* network_channel */ tx_network,
-            commit_channel,
-        );
-        tokio::spawn(async move {
-            core.run().await;
-        });
+        sleep(Duration::from_millis(parameters.timeout_delay)).await;
```

Comment:
When running the experiment locally, I found sometimes the first node will send the block before connecting to other nodes. So I added this period for all nodes to connect before running the consensus. Let me know if anything is wrong.

Reply:
That is very strange and is worth investigating. If the core boots before the network, it will simply fill the net mpsc channel (which will be consumed by the network as soon as it boots). If the problem is due because of some nodes booting before others, we should probably implement a similar trick to the bench client (https://github.com/asonnino/hotstuff/blob/5261b26c073ed67f2a69e5249322309bcc17bb93/node/src/client.rs#L157)